### PR TITLE
done

### PR DESCRIPTION
**Pull request by aionova**
[DEMO LINK](https://aionova.github.io/react_static-list-of-todos/)

### CODE REVIEW

**Review by OleksandrMykoliuk — APPROVED** [review]
Крутий дизайн. Супер!

File: src/App.js [hunk 1]
```diff
@@ -3,20 +3,30 @@ import './App.scss';
...
 
 function App() {
   return (
     <div className="App">
-      <h1>Static list of todos</h1>
-      <p>
+      <h1 className="App__head">Static list of todos</h1>
+      <p
+        className="App__count"
+      >
```
Comment (aliceblunt8) [inline src/App.js:21]:
```suggestion
      <p className="App__count">
```

File: src/components/Todo/Todo.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    justify-content-between
+    list-group-item
+    list-group-item-action
+  "
+  >
+    <div>
+      <p className="todo__title">
+        {title}
+      </p>
+      <i>User:</i>
```
Comment (aliceblunt8) [inline src/components/Todo/Todo.js:19]: try not to use i and b tags in the markup 

https://www.w3.org/International/questions/qa-b-and-i-tags

**Review by aliceblunt8 — APPROVED** [review]
gj

**Pull request closed without merging**